src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -2630,5 +2630,202 @@ static void Main()
                 Diagnostic(ErrorCode.ERR_BadSKknown, "System").WithArguments("System", "namespace", "type").WithLocation(6, 17)
                 );
         }
+
+        [Fact]
+        public void DefaultNullableParameter()
+        {
+            var text = @"
+class C
+{
+    static void Main() { A(); B(); D(); }
+
+    static void A(int? x = default) => System.Console.Write(x.HasValue);
+    static void B(int? x = default(int?)) => System.Console.Write(x.HasValue);
+    static void D(int? x = default(byte?)) => System.Console.Write(x.HasValue);
```

Comment:
>default(byte?) [](start = 27, length = 14)

This doesn't look like the scenario I suggested. I suggested default(**byte**).

Reply:
This scenario is interesting too, let's keep it.

---
In reply to: [143833945](https://github.com/dotnet/roslyn/pull/22527#discussion_r143833945) [](ancestors = 143833945)

Reply:
Ah, sorry about that. I'll add.